packages/api/src/context.ts

```diff
@@ -296,8 +305,15 @@ export async function contextFromRequest(
               console.error(peer)
               return null
             }
-
-            const fetcher = createFetcher(url.resolve(peer.hostURL, 'admin'), peer.token)
+            // TODO fix types
+            const peerTimeout = Number(
```

Comment:
don't we want to do a parseInt/+ here?

Reply:
Regarding type fix. I'm not sure if there's a way to handle how the expected type behaves here. It would always be a number but I get the warning because its expecting unknown. Is there a cleaner way to handle this?

Reply:
If it's already coming as number you should type it as one (value as number). parseInt, + and Number all have runtime changes.

using Number() has the side effect that strings that look like other bases, will not be in base 10. So parseInt/+ is preferred.

Reply:
And helpful to know about parseInt vs Number() :+1: